include/boost/json/string.hpp

```diff
@@ -3210,12 +3306,34 @@ class string
 
     //------------------------------------------------------
 
+    /** Change the size of the string.
+        
+        Resizes the string to contain `count` characters. If
+        `count > size()`, characters with the value `0` are
+        appended. Otherwise, `size()` is reduced to `count`.
+
+        @param count The size to resize the string to.
+
+        @throw std::out_of_range `count > max_size()`
+    */
     void
     resize(std::size_t count)
     {
         resize(count, 0);
     }
 
+    /** Change the size of the string.
+
+        Resizes the string to contain `count` characters. If
+        `count > size()`, copies of `ch` are
+        appended. Otherwise, `size()` is reduced to `count`.
+
+        @param count The size to resize the string to.
+        @param ch The characters to append if the size
+        increases.
+
+        @throw std::out_of_range `count > max_size()`
```

Comment:
Complexity and exception safety?

Reply:
Yup, I forgot the exception safety bit and already added it, just have not pushed the commit yet